app/views/layouts/application.html.erb

```diff
@@ -19,8 +19,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
 
     <% if ENV["GOOGLE_ANALYTICS_ID"] %>
-      <%= javascript_include_tag "https://www.googletagmanager.com/gtag/js?id=#{ENV["GOOGLE_ANALYTICS_ID"]}" %>
       <%= javascript_include_tag "google_analytics", data: { "ga-id" => ENV["GOOGLE_ANALYTICS_ID"] } %>
+      <%= javascript_include_tag "analytics" %>
```

Comment:
This should come first. The `DOMContentLoaded` event of `google_analytics` expects `ga` to exist and be a function, which won't be true until after `analytics` has loaded, and we don't really have any control over how GA sets itself up.